discovery-provider/plugins/verified-uploads/src/handlers/users.js

```diff
@@ -5,12 +5,17 @@ import axios from 'axios'
 import { getPreviousState } from './utils.js'
 
 dotenv.config()
-const { IDENTITY_URL, USERS_SLACK_CHANNEL } = process.env
+const { audius_discprov_identity_service_url, USERS_SLACK_CHANNEL } =
+  process.env
 const social_handle_url = (handle) =>
-  `${IDENTITY_URL}/social_handles?handle=${handle}`
+  `${audius_discprov_identity_service_url}/social_handles?handle=${handle}`
```

Comment:
is this right?

Reply:
yep! took it straight from stage.env
https://github.com/AudiusProject/audius-docker-compose/blob/stage/discovery-provider/stage.env#L20

Reply:
https://identityservice.staging.audius.co/social_handles?handle=totallynotalec